src/webpack/plugin/SitePlugin.js

```diff
@@ -59,128 +59,145 @@ class SitePlugin {
     sourceFolder,
     distributionFolder,
     basename,
+    isWebpack5,
   }) {
     if (oneTimeSetupComplete) {
       return;
     }
     oneTimeSetupComplete = true;
+    const mdxLoader = getMdxLoader(compiler.options.output.publicPath);
 
-    compiler.options.entry = {
-      ...compiler.options.entry,
-      rewriteHistory: '@cerner/terra-dev-site/lib/browser-router-redirect/rewriteHistory',
-      redirect: '@cerner/terra-dev-site/lib/browser-router-redirect/redirect',
-    };
+    // If a mono repo, update the rootDirectories to include all the packages.
+    const rootDirectories = [
+      ...isLernaMonoRepo ? [path.resolve(processPath, 'packages', '*')] : [processPath],
+    ];
 
-    // MODULE
-    const mdxLoader = getMdxLoader(compiler.options.output.publicPath);
-    compiler.options.module.rules = [{
-      // Drop loaders in a 'one of' block to avoid the original loaders applying on top of the new loaders.
-      // Only the first loader will apply and no others.
-      oneOf: [{
-        test: /\.mdx$/,
-        use: [
-          babelLoader,
-          mdxLoader,
-        ],
-      }, {
-        test: /\.md$/,
-        oneOf: [
-          {
-            // Use MDX to import any md files imported from an mdx file.
-            issuer: [
-              /\.mdx?$/,
-              /entry\.template$/,
+    let webpackConfig = {
```

Comment:
we now generate the defaults we want to apply as a webpack config, run it through a normalizer, then apply it to the options object.